src/nodejs/lambda-handlers/data.js

```diff
@@ -42,6 +42,15 @@ async function findAll({ resource, params, context }) {
   return db[resource].findAll(params);
 }
 
+async function createForm({ resource, params, context }) {
```

Comment:
This location is fine for now. Eventually, we may want to have a separate `form.js` lambda handler for form related methods.